packages/discovery-provider/Dockerfile.dev

```diff
@@ -73,8 +73,6 @@ COPY requirements.txt requirements.txt
 RUN --mount=type=cache,target=/root/.cache \
   python3 -m pip install -r requirements.txt --no-cache-dir
 
-COPY . .
```

Comment:
Can you just clarify _why_ it's unnecessary? I would assume we need at least some of these files for building/running. Are we getting them via a mount instead?

Reply:
yarp exactly